### PR TITLE
Devex - consistency for connect props

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/views/CaseDetail/ActionRequired.jsx [hunk 1]
```diff
@@ -21,9 +21,6 @@ const MailPayment = () => (
 
 export const ActionRequired = connect(
   {
-    caseDetail: state.formattedCaseDetail,
-    caseHelper: state.caseDetailHelper,
-    setDocumentDetailTabSequence: sequences.setDocumentDetailTabSequence,
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetail/ActionRequired.jsx:26]: 🗑 nice

File: web-client/src/views/CaseDetailReadOnly.jsx [hunk 2]
```diff
@@ -5,34 +5,32 @@ import React from 'react';
 
 export const CaseDetailReadOnly = connect(
   {
-    caseDetail: state.formattedCaseDetail,
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailReadOnly.jsx:8]: whoah, we brought in `formattedCaseDetail` with two different names?  *facepalm*

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/CaseDetailReadOnlyPartyInformation.jsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ export const CaseDetailReadOnlyPartyInformation = connect(
         <h3>Party Information</h3>
         <div>
           <span className="label">Party Type</span>
-          <p>{caseDetail.partyType || 'My Party Type'}</p>
+          <p>{formattedCaseDetail.partyType || 'My Party Type'}</p>
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailReadOnlyPartyInformation.jsx:47]: I'm wondering whether this default value is necessary -- and if so, perhaps it should be in the computed?

**Review by wbyoko — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Review by ericsorenson — APPROVED** [review]

**Pull request merged**